addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1477,12 +1520,19 @@ var SnippetsMenu = Widget.extend({
      * @param {Object} [style]
      */
     _insertDropzone: function ($hook, vertical, style) {
+        const forbidSanitize = this._insertDropzoneForbidSanitize && $hook.closest('[data-oe-sanitize]').length;
         var $dropzone = $('<div/>', {
-            'class': 'oe_drop_zone oe_insert' + (vertical ? ' oe_vertical' : ''),
+            'class': 'oe_drop_zone oe_insert' + (vertical ? ' oe_vertical' : '') +
```

Comment:
Probably still a WIP and to be adapted but

![image](https://user-images.githubusercontent.com/30048408/165787268-d6247dd3-3ae1-4872-88f9-a142fb2ce4bb.png)

=> 

![image](https://user-images.githubusercontent.com/30048408/165787407-4e99de09-621e-4e03-8aaa-39447dffc482.png)

I am not sure how is the regular dropzone actually blinking in a very light blue-ish background, but use a very light red-ish background instead, not that yellow :p

Reply:
Replaced `bg-warning` by `bg-danger`, and overloaded it in drop zones to make its alpha 15%.